src/main/resources/assets/js/application/recruiter/directives/tdprPersonsDirective.js

```diff
@@ -0,0 +1,82 @@
+define(['angular', 'application/recruiter/tdprRecruiterModule'], function (angular, tdprRecruiterModule) {
+    tdprRecruiterModule.directive("personsDirective", function () {
+        return {
+            restrict: 'A',
+            templateUrl: 'js/application/recruiter/views/tdpr-directive-persons.html',
+            scope: {
+                selectedPersonTable: '=',
+                timeElements: '=',
+                startWeekDay: '='
+            },
+            link: function (scope, element, attributes) {
+                scope.activePerson = null;
+                scope.AvailabilityEnum = Object.freeze(
```

Comment:
this could be stored in a separate file - even in a separate module (like tdprConstants)